saos-webapp/src/main/webapp/WEB-INF/view/analysis/globalContext/judgmentDateRangeFormSection.jsp

```diff
@@ -23,4 +23,8 @@
            </div>
 
       </div>
+      
+      <div class="align-right">
+          <button class="btn btn-sm button confirm-section-button"><spring:message code="confirm" /></button>
```

Comment:
Can't see the confirm-section-button in the css. Is it only for jquery selector purposes? If so then it's not the best solution. It's better to give every button a name or id starting with a common phrase.